README.md

```diff
@@ -1,284 +1,37 @@
-    > cmake -G "Visual Studio 14 <Win64>" <-DBOOST_LIBRARYDIR="<Boost libs path>>" ..
-    > msbuild ALL_BUILD.vcxproj </p:Configuration=<CFG>>
-
-**Note**: Add `Win64` to the `-G` option to build a 64-bit version of the driver.
-
-**Note**: Optionally select the location of the Boost libraries with the `-DBOOST_LIBRARYDIR` option.
-
-**Note**: Optionally select the build configuration with the `/p:Configuration=` option. Typically `Debug`, `Release`, `MinSizeRel` and `RelWithDebInfo` are available.
-
-##### Examples
-
-Building for with 64-bit Visual Studio 2015:
-
-    > cmake -G "Visual Studio 14" ..
-
-#### Ubuntu Linux
-
-Install cmake:
-
-    $ sudo apt-get install cmake
-
-Then change to the root folder of the repository and issue the following commands:
-
-    $ cd build
-    > cmake -G "Unix Makefiles" <-DCMAKE_BUILD_TYPE=<build_type>> <-DARCH=<x86_32,x86_64>> <-DBOOST_LIBRARYDIR="<Boost libs path>>" ..
-    $ make
-
-**Note**: Optionally Select the build configuration with the `-DCMAKE_BUILD_TYPE` option. Typically `Debug`, `Release`, `MinSizeRel` and `RelWithDebInfo` are available.
-
-**Note**: Optionally select the target architecture (32 or 64-bit) using the `-DARCH` option.
-
-**Note**: Optionally select the location of the Boost libraries with the `-DBOOST_LIBRARYDIR` option.
-
-#### macOS (OS X) 10.11 and later
-
-Install cmake with Homebrew with the `brew` command on a terminal:
-
-    $ brew install cmake
-
-Then change to the root folder of the repository and issue the following commands:
-
-    $ cd build
-    $ cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE= <build_type> ..
-    $ make
-
-**Note**: Optionally Select the build configuration with the `-DCMAKE_BUILD_TYPE` option. Typically `Debug`, `Release`, `MinSizeRel` and `RelWithDebInfo` are available.
-
-#### Running the examples
-
-Now that you have successfully built `pc-ble-driver`, you are ready to run the examples in `pc-ble-driver/examples`. Verify that the static and shared libraries exist in the directory the examples expect them to be in.
+Now that you have successfully built and installed `pc-ble-driver`, you are ready to run the examples in `pc-ble-driver/examples/`. First verify that the static and shared libraries exist in the directory the examples expect them to be in.
 
     $ cd pc-ble-driver/
     $ ls build/
     > libpc_ble_driver_static_sd_api_v2.a libpc_ble_driver_shared_sd_api_v2.dylib libpc_ble_driver_static_sd_api_v3.a  libpc_ble_driver_shared_sd_api_v3.dylib test_uart ...
 
 To quickly get the examples up and running, see [examples/README.md](https://github.com/NordicSemiconductor/pc-ble-driver/blob/master/examples/README.md).
+    
+## Architecture
+
+![alt tag](http://infocenter.nordicsemi.com/topic/com.nordic.infocenter.sdk5.v13.0.0/architecture_overview_serialization.svg)
+
+Where the 'Application chip' is just generic hardware (i.e. a Windows, OS X or Linux device), although it could also be an Arduino or Raspberry Pi for example.
```

Comment:
Since we are in context of pc-ble-driver here it's not true to claim that it will run on Arduino. Serlialization in general, yes, but not pc-ble-driver. Raspberry Pi is ok, because that actually runs Linux and can compile pc-ble-driver.